youtube_dl/extractor/olympicchannel.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, try_get, unified_strdate
+
+
+class OlympicChannelIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?olympicchannel\.com/../playback/(?P<id>.+)'
```

Comment:
Also looks like they redesigned the site so that extractor does not work any longer.

Reply:
Updated for new layout. There is still a `<div id='ocs-video' data-options='...'>`, but it no longer contains all the metadata (missing description). However, they added new meta fields with everything we need, so I went with those instead which simplified things a bit.